### DISCUSSION TITLE
v0.9.0-rc2

### DISCUSSION THREAD

**Discussion by lukfor** [askimed/nf-test#220] [Announcements]
**⚠️ This is a pre-release and is not yet production ready.**

## What's Changed
* Add flag to filter dependencies based on test case type by @lukfor in https://github.com/askimed/nf-test/pull/219

**Full Changelog**: https://github.com/askimed/nf-test/compare/v0.9.0-rc1...v0.9.0-rc2

<hr /><em>This discussion was created from the release <a href='https://github.com/askimed/nf-test/releases/tag/v0.9.0-rc2'>v0.9.0-rc2</a>.</em>

**Comment by sateeshperi** [comment]
hi @lukfor any idea when `0.9` is gonna be released ?

**Comment by lukfor** [comment]
Hi! I hope the end of this week... semester is coming to an end ;)
